bigframes/core/blocks.py

```diff
@@ -1570,6 +1575,52 @@ def melt(
             index_columns=[index_id],
         )
 
+    def transpose(self) -> Block:
+        """Transpose the block. Will fail if dtypes incompatible or too many rows"""
```

Comment:
Suggest we change "dtypes incompatible" to "dtypes cannot be coerced all to a common type."

Reply:
done